src/config/environments/development.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # BEGIN: needed for email confirmations
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { address: 'localhost', port: 1025 }
```

Comment:
Why doesn't the SMTP server run on port 25? (Specifically, if this is running in a virtual env I think everything should run on stock default ports.) Or, maybe there is something about smtp servers running on port 1025 that I'm missing?!